src/popup/router/components/Modal.vue

```diff
@@ -49,6 +49,12 @@
           <slot />
         </div>
 
+        <div
+          v-if="$slots.customBody"
+        >
+          <slot name="customBody" />
+        </div>
+
```

Comment:
To control the default body padding two props are available: `dense` or `noPadding`. Please use one of them.